beets/util/__init__.py

```diff
@@ -432,14 +448,14 @@ def syspath(path, prefix=True):
     return path
 
 
-def samefile(p1, p2):
+def samefile(p1: Bytestring, p2: Bytestring) -> bool:
     """Safer equality for paths."""
     if p1 == p2:
         return True
     return shutil._samefile(syspath(p1), syspath(p2))
 
 
-def remove(path, soft=True):
+def remove(path: AnyStr, soft: bool = True):
```

Comment:
`path: bytes` (same for the functions below)

Reply:
Fixed in 5abe5d54